app/models/conference.rb

```diff
@@ -12,12 +12,16 @@ class Conference < ActiveRecord::Base
   scope :ordered, ->(sort = {}) { order([sort[:order] || 'starts_on, name', sort[:direction] || 'asc'].join(' ')) }
   scope :in_current_season, -> { where(season: Season.current) }
 
+  def dates
```

Comment:
Ah yes, would make good sense here I think 👍 
The method is used several times while rending some views and it's unlikely that start or end will change (and require recomputation) in between.

Good idea!